packages/grid/src/utils/types.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export type TYPE_ALIGN_ITEMS = 'start' | 'end' | 'center' | 'baseline' | 'stretch';
```

Comment:
For naming in other packages, we've used ALL_CAPS to denote types. It may be more readable to drop the `TYPE_` prefix.

If you use the `F2` option in VSCode it will rename all usages for you 😄

Reply:
But as you can see here, it's meant to differentiate from the `ARRAY` constant.

Reply:
Are you suggesting to rename all the `type` constants and leave the array naming as-is?

Reply:
> Are you suggesting to rename all the type constants and leave the array naming as-is

That could make sense. We just don't normally define arrays for the `propType` definitions in other components. If this feels right to you feel free to leave it.